examples/models/llama2/source_transformation/hqq_16a4w.py

```diff
@@ -0,0 +1,205 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
maybe you want to move this to torchao: https://github.com/pytorch/ao/tree/main/torchao/prototype/hqq?